app/src/main/scala/org/alephium/explorer/api/AddressesEndpoints.scala

```diff
@@ -113,7 +113,7 @@ trait AddressesEndpoints extends BaseEndpoint with QueryParams {
   val listAddressTokens: BaseEndpoint[(Address, Pagination), ArraySeq[TokenId]] =
     addressesTokensEndpoint.get
       .out(jsonBody[ArraySeq[TokenId]])
-      .in(paginator(defaultLimit = 100))
+      .in(paginator(limit = 100))
```

Comment:
Does it mean the default limit is 100 for tokens?

Reply:
yes, this endpoint is just returning `TokenId` stored in one table, no heavy query like tx. But we could reduce it too if you prefer.

Reply:
maybe run some benchmark on this to see the latency

Reply:
Reading 100 or 10 row from DB is equivalent. 
![image](https://github.com/user-attachments/assets/2831e176-c541-4ebd-adb4-0f3cb37d64f2)
![image](https://github.com/user-attachments/assets/ca0e3ea5-5f39-422b-b1f4-67b9fe25027d)

Reply:
Interesting. Then let's keep it unchanged for now